docs/tool_jextract.md

```diff
@@ -30,14 +30,21 @@ On some operating systems, copies of executable files and libraries are required
 
 ## Syntax
 
-        jextract <core file name> [<zip_file>]
+        jextract [-r] <core file name> [<zip_file>]
 
 where:
 
+- `-r` forces the `jextract` utility to proceed when the system dump is created from an SDK with a different build ID. See **Restriction**.
 - `<core file name>` is the name of the system dump.
 - `<zip_file>` is the name you want to give to the processed file. If you do not specify a name, output is written to `<core file name>.zip` by default. The output is written to the same directory as the core file.
 
-: <i class="fa fa-exclamation-triangle"></i> **Restriction: ** You must run `jextract` on the same system that produced the system dump in order to collect the correct executables and libraries referenced in the system dump. You should also run `jextract` using the same VM level, to avoid any problems.
+: <i class="fa fa-exclamation-triangle"></i> **Restriction:** You should run `jextract` on the same system that produced the system dump in order to collect the correct executables and libraries referenced in the system dump. You should also run `jextract` using the same VM level, to avoid any problems. From Eclipse OpenJ9 V0.23.0, the utility always checks that the build ID of the SDK that created the dump file matches the `jextract` build ID. Where these IDs do not match, the following exception is thrown:
```

Comment:
I don't see the exclamation in the staging view; will it appear in the final version?
This section appears indented as if it were part of the '<zip_file>' bullet point - I don't think it should be indented.

Reply:
Still working on the indentation @keithc-ca - it can be painful with markdown, especially after the extension (:) we are using. 

The missing exclamation triangle is a bug we are attempting to fix in a separate thread. The font awesome icons are missing generally, not just here.

Reply:
Indentation now fixed on staging. The icon will turn back on when we fix the general problem. If the content is now good we should get this merged.